docs/axes/cartesian/README.md

```diff
@@ -54,11 +55,10 @@ var myChart = new Chart(ctx, {
     },
     options: {
         scales: {
-            yAxes: [{
-                id: 'first-y-axis',
+            'first-y-axis': {
```

Comment:
I'm assuming this syntax works though it's a bit awkward as an example. Maybe `firstYAxis` would be easier since it wouldn't require the single quotes